src/Cms/App.php

```diff
@@ -1307,7 +1307,7 @@ public function resolve(string $path = null, string $language = null)
 		$extension = F::extension($path);
 
 		// no content representation? then return the page
-		if (empty($extension) === true) {
+		if (empty($extension) === true && Str::endsWith($path, '.') === false) {
```

Comment:
With this change, the representation lookup logic will continue for such paths with a trailing dot. So Kirby would try to find a template like `default..php`.

Do we explicitly want to support this? If not, I think it would be better if we instead used `if (Str::endsWith($path, '.') === true) { return null; }` already pretty much at the top of the method for optimal performance.

Reply:
I think you are right, I don't think it makes much sense to support basically an empty representation type with `default..php`